content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -112,6 +127,17 @@ def requested_number_of_cpus(args_hash)
                    get_option_value(args_hash[:resource], :cores_per_socket)
   cpu_in_request = get_option_value(args_hash[:resource], args_hash[:number_of_cpus]) if cpu_in_request.zero?
   args_hash[:prov_value] = args_hash[:number_of_vms] * cpu_in_request
+
+  if @reconfigure_request && args_hash[:resource].options[:number_of_sockets]
+    # Account for the VM's existing CPUs
+    args_hash[:prov_value] = args_hash[:prov_value].to_i - @vm.hardware.cpu_total_cores.to_i \
+      * @vm.hardware.cpu_cores_per_socket.to_i
+
+    $evm.log(:info, "vm_number_of_cpus:         #{@vm.hardware.cpu_total_cores.to_i \
+      * @vm.hardware.cpu_cores_per_socket.to_i}")
+    $evm.log(:info, "requested_number_of_cpus:  #{args_hash[:prov_value].to_i}")
+    @bypass_quota = false if args_hash[:prov_value].to_i > 0
```

Comment:
@billfitzgerald0120 @tinaafitz 
The bypass_quota seems like an incorrect variable name here. Should it rather be
```
 @check_quota = true if args_hash[:prov_value].to_i > 0
```

If its a reconfigure request we can set the
```
@check_quota = false
```
And have one of these methods enable it, if the memory, cpu, storage in reconfigure request is greater than the currently allocated amount.